robots/release/release_members.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Robot class to run under multiplexing infrastructure
+module Robots
+  module DorRepo
```

Comment:
similar question about directory structure adhering to implied containment in code.  Here, important only if preservation robots come in.  (TBH, I hate the "sdr" vs "dor" in the robots and would be happy for it to go away)

Reply:
This is just a straight copy from https://github.com/sul-dlss/item-release/blob/master/robots/release/release_members.rb